rclpy/test/test_node.py

```diff
@@ -587,6 +587,11 @@ def reject_parameter_callback(self, parameter_list):
         rejected_parameters = (param for param in parameter_list if 'reject' in param.name)
         return SetParametersResult(successful=(not any(rejected_parameters)))
 
+    def reject_parameter_callback_1(self, parameter_list):
+        rejected_parameters = (
+            param for param in parameter_list if 'accept' in param.name)
```

Comment:
Rejecting parameter names that contain "accept" seems counter-intuitive and might confuse future developers.

I think what you had here before was fine.

Reply:
@jacobperron sir the the previous I used was "reject1" but this was rejecting the parameter with name "reject" any name which contains the string "reject" so I used different a total different name,far from "reject",otherwise both will be same then isnt it?